aws-samples/amazon-chime-sdk
# Add Amazon Chime Echo Reduction for React SDK

**Issue #:**

**Description of changes:**
Add Amazon Chime Echo Reduction for React SDK
**Testing**

1. How did you test these changes?
Testing steps:
- Enable Web Audio. Enable Echo Reduction. Join the meeting
- Select one input audio device, and then click on "Enable voice focus".
- Expect: ns_es model get download and exactly once.
2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
apps/meeting
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
